app/controllers/friend_requests_controller.rb

```diff
@@ -20,13 +14,7 @@ def current_user_sent_friend_requests
   end
 
   def current_user_received_friend_requests
```

Comment:
same goes here

Reply:
current_user_received_friend_requests (GET request) here means that the request I received from other users.